src/main/java/Transcendent.java

```diff
@@ -0,0 +1,178 @@
+import java.util.Scanner;
+
+public class Transcendent {
+
+    public static void main(String[] args) {
+        Printer.printEntry();
+        List.init();
+        InputHandler.takeInput();
+        Printer.printExit();
+    }
+
+    private static class Printer {
+
+        private static void printEntry() {
+            Printer.printSeparator();
+            System.out.println("Greetings.");
+            System.out.println("I am Transcendent.");
+            System.out.println("What do you need assistance with?");
+            Printer.printSeparator();
+        }
+
+        private static void printSeparator() {
```

Comment:
good job making the printSeparator() as a separate method to cut down on repeating it